ext/LICENSE

```diff
@@ -0,0 +1 @@
+Copyright 2019 Kenneth Reitz. All rights reserved.
```

Comment:
**Note**: this is for the images contained within the `ext` folder, which includes a [tattoo upon my arm](https://images.squarespace-cdn.com/content/v1/533ad9bde4b098d084a846b1/1469928300857-S9WKXBO0CCRX0RL4PJYX/ke17ZwdGBToddI8pDm48kKLHmMs23ZxPgeOu8Yrs55d7gQa3H78H3Y0txjaiv_0fDoOvxcdMmMKkDsyUqMSsMWxHk725yiiHCCLfrh8O1z4YTzHvnKhyp6Da-NYroOW3ZGjoBKy3azqku80C789l0iyqMbMesKd95J-X4EagrgXXATmnXitsEasKvdtVty7KcIpiUOmkkpuOP9PWkdpi9w/L1000129-Edit.jpg?format=1500w) — I had the tattoo before I made it the Requests logo :)